NebulousConquestHelper/src/bot/BotCommands.cs

```diff
@@ -0,0 +1,25 @@
+﻿using DSharpPlus.CommandsNext;
+using DSharpPlus.CommandsNext.Attributes;
+using DSharpPlus.Entities;
+using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace NebulousConquestHelper.src.bot
```

Comment:
I think all the C# classes in the subproject should be under the NebulousConquestHelper namespace, and not split into several namespaces by folder. Open to changing that though. If no objections, can you change it for this file and BotHelpFormatter?

Reply:
Yup! I didn't explicitly set this namespace. It was auto created when adding the files. I'll make sure to clear that up with future additions.

Reply:
Namespaces corrected